.changelog/20251121165253_ck_19138.md

```diff
@@ -0,0 +1,45 @@
+---
+# Required: Type of change.
+# Allowed values:
+# - Feature
+# - Fix
+# - Other
+# - Major breaking change
+# - Minor breaking change
+#
+# For guidance on breaking changes, see:
+# https://ckeditor.com/docs/ckeditor5/latest/updating/versioning-policy.html#major-and-minor-breaking-changes
+type: Feature
+
+# Optional: Affected package(s), using short names.
+# Leave empty when used in a single-package repository.
+# Example: ckeditor5-core
+scope:
+  - 
+
+# Optional: Issues this change closes.
+# Format:
+# - {issue-number}
+# - {repo-owner}/{repo-name}#{issue-number}
+# - Full GitHub URL
+closes:
+  - 
+
+# Optional: Related issues.
+# Format:
+# - {issue-number}
+# - {repo-owner}/{repo-name}#{issue-number}
+# - Full GitHub URL
+see:
+  - 
+
+# Optional: Community contributors.
+# Format:
+# - {github-username}
+communityCredits:
+  - 
+
+# Before committing, consider removing all comments to reduce file size and enhance readability.
+---
+
+`generateChangelogForMonoRepository()` will now provide examples based on the actual current version when asking user about release type.
```

Comment:
This change applies to both changelog methods: `generateChangelogForSingleRepository() and `generateChangelogForMonoRepository()`.